client/src/lib/components/wallet/wallet-balance.svelte

```diff
@@ -26,7 +26,10 @@
     const adjustedBalance = BigInt(balance) / factor;
     const formattedBalance = Number(adjustedBalance) / 10 ** precision;
 
-    return formattedBalance.toPrecision(precision);
+    return formattedBalance.toLocaleString('en-US', {
+      minimumFractionDigits: precision,
+      maximumFractionDigits: precision,
```

Comment:
```suggestion
      minimumFractionDigits: 4,
      maximumFractionDigits: 4,
```
This is the value in Argent, this might need an update if we have a really strong token

Reply:
Nevermind